src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -17,6 +17,7 @@
     public CommandResult executeUndoableCommand() {
         requireNonNull(model);
         model.resetData(new AddressBook());
+        model.unselect();
```

Comment:
What is this? oO why unselect when execute undoableCommand()? Oo

Reply:
The model will still have the flag "selected" if we don't switch it back.